test/functionalTest/cases/1433_update_metadata_duplicated/update_meta.test

```diff
@@ -0,0 +1,188 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Update metadata changing type, not duplicated
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# Issue #1713 reports that NULL value for metadatas isn't working.
+#
+# 01. Create a metadata
+# 02. GET the entity to make sure the creation worked
+# 03. Update metadata changing type
+# 04. GET the entity to make sure the creation worked
```

Comment:
A little more info in the 'toc' would be nice, such as:

01. Create a metadata with type 'xxx'
02
03. Update metadata, changing type to null